openfeature/api.py

```diff
@@ -39,6 +39,12 @@ def get_client(
     return OpenFeatureClient(domain=domain, version=version)
 
 
+def get_client_async(
```

Comment:
```suggestion
def get_async_client(
```
I would prefer this name, because the current one somehow implies I'm getting the client asynchronously, at least for me 😅

Reply:
Agreed, a bit of an oversight.